srtcore/core.cpp

```diff
@@ -2634,7 +2639,7 @@ bool CUDT::interpretSrtHandshake(const CHandShake& hs,
                     return false;
                 }
                 handshakeDone();
-                updateAfterSrtHandshake(SRT_CMD_HSREQ, HS_VERSION_SRT1);
+                // updateAfterSrtHandshake -> moved to postConnect and processRendezvous
```

Comment:
Temporal hint for merging?

Reply:
Yes. After some time I myself don't remember why the line is changed, and additionally, automatic merge resolved tend to simply restore removed lines without telling me anything. This at least causes an irresolvable conflict that I will resolve the right way.